bundle/manifests/discovery.clusterserviceversion.yaml

```diff
@@ -8,7 +8,8 @@ metadata:
           "apiVersion": "discovery.open-cluster-management.io/v1alpha1",
           "kind": "DiscoveryConfig",
           "metadata": {
-            "name": "discovery"
+            "name": "discovery",
+            "namespace": "open-cluster-management"
```

Comment:
@JakobGray does this matter and/or should it go away?

Reply:
That's fine the way it is. It's an example CR embedded in the CSV.

Reply:
roger